### PR TITLE
Add "Code" badge to code-configured data sources

### PR DESCRIPTION
**Pull request by chriszarate**
Add "Code" badge to code-configured data sources to help explain why their actions are disabled.

Screenshot 2024-12-26 at 12 30 17

### CODE REVIEW

File: src/data-sources/DataSourceMetaTags.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+				</>
+			) }
+		</span>
+	);
+};
+
+const DataSourceMetaTags = ( props: DataSourceMetaTagsProps ) => {
+	return (
+		<>
+			{ ! props.source.uuid && <span className="data-source-badge">Code</span> }
```
Comment (alecgeatches) [inline src/data-sources/DataSourceMetaTags.tsx]: What do you think about a `Tooltip` (from `@wordpress/components`) here with some more information?

```suggestion
			{ ! props.source.uuid && <Tooltip text={ __( 'This source is configured in code', 'remote-data-blocks' ) }><span className="data-source-badge">Code</span></Tooltip> }
```

The result looks like this:

Screenshot 2024-12-26 at 2 27 09 PM

**Review by alecgeatches — APPROVED** [review]
This looks great! Commented a suggestion above for adding a `Tooltip` with a bit of an explanation.

File: src/data-sources/DataSourceMetaTags.tsx (hunk 1 above)
Comment (chriszarate) [reply src/data-sources/DataSourceMetaTags.tsx]: I dig it.

Comment (chriszarate) [reply src/data-sources/DataSourceMetaTags.tsx]: Done in https://github.com/Automattic/remote-data-blocks/pull/264/commits/86384cbb06325417fd7c8391420d5460c0f269d6

**Pull request merged**